packages/components/src/query-controls/README.md

```diff
@@ -32,11 +32,21 @@ const QUERY_DEFAULTS = {
 	numberOfItems: 10,
 	order: 'asc',
 	orderBy: 'title',
+	orderByOptions: [
+		{
+			label: 'Title Ascending',
+			value: 'title/asc',
+		},
+		{
+			label: 'Title Descending',
+			value: 'title/desc',
+		},
+	],
 };
 
 const MyQueryControls = () => {
 	const [ query, setQuery ] = useState( QUERY_DEFAULTS );
-	const { category, categories, maxItems, minItems, numberOfItems, order, orderBy  } = query;
+	const { category, categories, maxItems, minItems, numberOfItems, order, orderBy } = query;
```

Comment:
@Mamaduka I got confused by the use of `categories` here. Technically that is also not part of the query coming from block attributes. How do you suggest we represent the `orderByOptions` here, if at all?

Reply:
The new options are included now in `QueryControls` types, which should be a good hint for developers.

We should also update the `categories` handler, but that's for a different PR.